### PR TITLE
data: AliMora83

### PR DESCRIPTION
**Pull request by AliMora83**
## Fixes Issue

N/a

## Changes proposed

- Updated Twitter icon.
- Made icons white.
- Restructured tags to = MERN.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Icon change and color will be white.
Screenshot 2023-05-20 at 10 42 08

Adjusted to MERN first.
Screenshot 2023-05-20 at 11 12 56

## Note to reviewers

I'm enjoying working with other developers 🥳
Thank you sooo much 🥳🙏🏼🤩

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7024"></a>

### CODE REVIEW

File: data/AliMora83.json [hunk 1]
```diff
@@ -20,13 +21,15 @@
       "group": "Socials",
       "name": "Follow me on LinkedIn",
       "url": "https://www.linkedin.com/in/daurius-ali-swarts-1b151420b/",
-      "icon": "FaLinkedin"
+      "icon": "FaLinkedin",
+      "iconColor": "white"
```
Comment (amandamartin-dev) [inline data/AliMora83.json:25]:
```suggestion
      "color": "white"
```

File: data/AliMora83.json [hunk 2]
```diff
@@ -20,13 +21,15 @@
...
     },
     {
       "group": "Socials",
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/swarts_d",
-      "icon": "FaLinkedin"
+      "icon": "FaTwitter",
+      "iconColor": "white"
```
Comment (amandamartin-dev) [inline data/AliMora83.json:32]:
```suggestion
      "color": "white"
```

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: data/AliMora83.json (hunk 1 above)
Comment (AliMora83) [reply data/AliMora83.json:25]: Thank you, will update 🥳

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**